src/main/java/pl/betse/beontime/timesheet/model/MonthDayBody.java

```diff
@@ -0,0 +1,38 @@
+package pl.betse.beontime.timesheet.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.validator.constraints.Length;
+
+import pl.betse.beontime.timesheet.validation.CreateTimeEntry;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.PositiveOrZero;
+import java.math.BigDecimal;
+
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class MonthDayBody {
+    @NotEmpty(groups = {CreateTimeEntry.class}, message = "Date can't be empty!")
+    @NotNull(groups = {CreateTimeEntry.class}, message = "Date can't be null!")
+    private String date;
+
+    @NotNull(groups = {CreateTimeEntry.class}, message = "Hours can't be null!")
+    @PositiveOrZero(groups = {CreateTimeEntry.class}, message = "Hours value should be equal or bigger than 0.")
+    @Max(groups = {CreateTimeEntry.class}, message = "Hours value should be equal or smaller than 24.", value = 24)
+    private BigDecimal hours;
```

Comment:
do we really need to keep hours in BigDecimal? It is quite complex object to store accurate number values

Reply:
I think there should be Double value and it will be enough for that purpose because users can only save values for example: 1, 2.5, 5.